proto/kvrpcpb.proto

```diff
@@ -226,3 +227,44 @@ message RawDeleteResponse {
     errorpb.Error region_error = 1;
     string error = 2;
 }
+
+message WriteInfo {    
+    uint64 start_ts = 1;
+    Op type = 2;
+    uint64 commit_ts = 3;
+    bytes short_values = 4;
+}
+
+message ValueInfo {
+    bytes value = 1;
+    uint64 ts = 2;
+}
+
+message MvccInfo {
+    LockInfo lock = 1;
+    repeated WriteInfo writes = 2;
+    repeated ValueInfo values = 3;
+}
+
+message MvccGetByKeyRequest {
+    Context context = 1;
+    bytes key = 2;
+}
+
+message MvccGetByKeyResponse {
+    errorpb.Error region_error = 1;
+    string error = 2;
+    MvccInfo info = 3;
+}
+
+message MvccGetByStartTsRequest {
+    Context context = 1;
+    uint64 start_ts = 2;
+}
+
+message MvccGetByStartTsResponse {
+    errorpb.Error region_error = 1;
+    string error = 2;
+    bytes key = 3;
+    MvccInfo info = 4;
```

Comment:
I know. But should `GetByTs` returns mvcc values of multiple keys that belong to the same txn?